### DISCUSSION TITLE
Can not use Copilot in VS2022

### DISCUSSION THREAD

**Discussion by Th30wl** [community/community#14239] [Copilot Conversations] [answered]
Visual Studio 2022 17.1.3

While installing the plugin i was signed in to the Visual Studio with account that does not have access to Copilot and now I can not use it. I have tried adding GitHub account to Visual Studio but it had no effect. 
Console displays the following:
```
GitHub Copilot 1.10.0.3 (v1.10.0.3@764d4f318)
GitHub Copilot Agent v1.10.0
You don't have access to GitHub Copilot.
```

Seems like the Copilot is trying to use the wrong account. Is there any way I can let the Copilot know which account to use?

In VS Code everything works fine.

**Comment by kzu** [comment, marked as answer]
Can you try signing out via the Copilot icon next to the editor zoom level? 

![image](https://user-images.githubusercontent.com/169707/161859274-71458285-0565-460d-8af9-ace453d58eb5.png)

If that doesn't work, you can delete the entire folder `%LOCALAPPDATA%\github-copilot` which will restore it to its initial state.

When you enter the authorization code on the https://github.com/login/device page, you will be authorizing with whichever account that browser window happens to be signed-in to GitHub. You can alternatively open an incognito window, navigate to that same page, sign-in to GitHub with a different account and paste the authorization code next.

**Reply by Th30wl** [reply]
That did the trick, thanks!

**Reply by kasunkoswattha-pg** [reply]
Any idea why its not giving me a device code? The output window keeps saying NotSignedIn
I was trying to manually enter the code but no code has been copied. 

![image](https://user-images.githubusercontent.com/101729345/164233294-a54afe5b-21aa-47f7-957f-4ab765d1cf8a.png)

**Reply by EhtuCom** [reply]
I did all the steps and looks that everything is fine, but no code completitions are shown and the GitHub Copilot window is empty:
I triet also the "trigger inline suggestion" in many different parts of code (controllers, classes, dbsets, etc)and nothing :S
![image](https://user-images.githubusercontent.com/4345616/175247679-1343eb02-d5a4-4396-91ed-b809b75ce0a8.png)

![image](https://user-images.githubusercontent.com/4345616/175247993-00bf4f36-973c-44e4-8215-9a8272542a5b.png)

**Reply by ObaidNiazi** [reply]
disable copilot - delete copilot cache folder - reinstall, 
can't remember who suggested it but it (kzu probably) worked for me. 
Thanks you so much

**Comment by LGinC** [comment]
![image](https://user-images.githubusercontent.com/15363011/162350969-16a6119f-2cca-4793-ace4-8feac5071a8e.png)
![image](https://user-images.githubusercontent.com/15363011/162351010-d4f7d401-5233-4b6c-b8a5-27e3e0bd95c3.png)
![image](https://user-images.githubusercontent.com/15363011/162351170-5b82981b-6c01-4ad5-8e21-0778671177f8.png)
![image](https://user-images.githubusercontent.com/15363011/162351212-2f736d85-7a34-40d3-aefb-9b74ba9681ff.png)
there is no code for device activation, and my clipboard is empty, `%LOCALAPPDATA%\github-copilot` is not existed.
I tried to restart visual studio and reinstall extension, but not work.
Visual Studio 2022 17.1.3

**Reply by LGinC** [reply]
when I turn off system proxy, there is a code but not init in `%LOCALAPPDATA%\github-copilot`,  cannot signIn
![image](https://user-images.githubusercontent.com/15363011/162353540-d1dbb198-244e-426b-af32-5f35635f20db.png)
![image](https://user-images.githubusercontent.com/15363011/162353517-ea0aea1f-52d1-497f-a63b-b66d1efed521.png)
![image](https://user-images.githubusercontent.com/15363011/162353575-cd34fd5a-bc13-45ff-8f9a-fbb46131c7ca.png)

**Reply by Elaina-Official** [reply]
请问为什么我无论是开启代理还是关闭代理都无法成功跳转到外部网站，我尝试过重启vs2022，但他仍然不起作用，去copilot图标sign in也一直只是显示NotSignedIn，怎么办。我在vscode上面使用copilot就没有任何问题，但是在vs上就没办法正常使用

**Reply by LGinC** [reply]
手动输入https://github.com/login/device  试试呢

**Reply by Elaina-Official** [reply]
前天登的时候突然又好了，已经成功用上了，不过感觉速度没vsc的快。估计是服务器的问题，好多人都登不上，不过还是谢谢了

**Reply by 12UE** [reply]
这个问题并没有解决，无论是开不开启代理都能正常访问浏览器的页面 但是我输入了验证码之后 页面上只是显示了“Congratulations, you're all set!” Vs2022 并没有做出任何反馈 也没有提示我登入成功或者失败 再次点击登入按钮之后 只会重复上面的步骤 没有任何反馈

This problem has not been solved. The browser page can be accessed normally whether the agent is turned on or not. However, after I enter the verification code, the page only displays "Congratulations, you're all set!" Vs2022 doesn't give any feedback or prompt me about success or failure of login. After clicking the login button again, the above steps will be repeated without any feedback.

**Reply by Elaina-Official** [reply]
你可以试一下重新登录Microsoft账户和GitHub账户，如果显示Congratulations那么就重新进入VS2022看看是否可以使用。

**Reply by 12UE** [reply]
遗憾的是，我按照你所描述的方法重新登入了vs2022上的github账户 ，问题依旧没有改变。

Unfortunately, I re logged into the GitHub account on vs200 according to the method you described. The problem remains unchanged.

**Reply by Elaina-Official** [reply]
如果确实是这样的话，请过一段时间在无代理状态下尝试。如果仍然不行，很遗憾我也帮不上忙。

**Reply by 12UE** [reply]
经过一波谷歌 找到了一个方法可用：
* 首先关闭你的所有VPN代理修改你的本地dns主服务器缓存为谷歌的8.8.8.8备用dns服务器使用阿里云的223.5.5.5
* 修改完以后点击键盘上的WIN + R 运行cmd
* 在键盘输入中输入ipconfig \flushdnsdns缓存
* 这个时候再打开你的visual studio 2022试试是否可以成功

**Reply by BillDong2021** [reply]
一样的问题，输入了验证码之后 页面上只是显示了“Congratulations, you're all set!” Vs2022 并没有做出任何反馈 也没有提示我登入成功或者失败 再次点击登入按钮之后 只会重复上面的步骤 没有任何反馈。
改8888ip的那个方法也不行，太难了，怎么办啊呜呜

Same！
This problem has not been solved. The browser page can be accessed normally whether the agent is turned on or not. However, after I enter the verification code, the page only displays "Congratulations, you're all set!" Vs2022 doesn't give any feedback or prompt me about success or failure of login. After clicking the login button again, the above steps will be repeated without any feedback.

**Reply by Elaina-Official** [reply]
> 一样的问题，输入了验证码之后 页面上只是显示了“Congratulations, you're all set!” Vs2022 并没有做出任何反馈 也没有提示我登入成功或者失败 再次点击登入按钮之后 只会重复上面的步骤 没有任何反馈。 改8888ip的那个方法也不行，太难了，怎么办啊呜呜
> 
> Same！ This problem has not been solved. The browser page can be accessed normally whether the agent is turned on or not. However, after I enter the verification code, the page only displays "Congratulations, you're all set!" Vs2022 doesn't give any feedback or prompt me about success or failure of login. After clicking the login button again, the above steps will be repeated without any feedback.

copilot现已需要付费使用...

**Reply by BillDong2021** [reply]
这方面没问题，就是GFW的问题，导致没法认证，账号是OK的

**Comment by lmiguelmh** [comment]
Does this extension work on the Community Edition? 
I did install the extension but nothing happens when opening VS.

**Reply by Elaina-Offcial** [reply]
Yes. Copilot can work on the Community Edition, im a Community Edition user.

**Reply by lmiguelmh** [reply]
Thanks for the answer... so there must some issue in my case. 
The extension was installed on Visual Studio 2022 (17.1.4) but I was never asked to enter the device code. 

![image](https://user-images.githubusercontent.com/3931964/164360730-1483b777-8045-4a86-b46c-ebcae2036db3.png)

---
Edit: Installation logs seems ok too:

> Install to Microsoft Visual Studio Community 2022 completed successfully. The extension has been installed to C:\USERS\...\APPDATA\LOCAL\MICROSOFT\VISUALSTUDIO\...\EXTENSIONS\MLOG4AIY.YIA\

**Reply by Elaina-Official** [reply]
If your case like this(Just repeated 'NotSignedIn' without Code):
![image](https://user-images.githubusercontent.com/69241368/164361302-f7862dfb-00ec-494c-b1ed-af00da13bd67.png)
You can try to wait for some time.
When Copilot is available on VS, I can't get the code as well, and I waited for a day. Then the next day I opened VS2022, the Code appeared. After that, Copilot can run correctly on VS2022. So I suggest you waiting for some time and then try again. Before that, please make sure that Copilot is authorized to your account.

**Reply by lmiguelmh** [reply]
Thank you. I will wait. 👍

**Reply by zhougx9981** [reply]
> Thank you. I will wait. 👍

May I ask if you have solved the problem？ I have the same problem。

**Comment by chesthar** [comment]
Visual Studio Pro 2022 17.2.5
Copilot 1.31

For people who still have this message "You don't have access to GitHub Copilot.", after login to your GitHub account and fill in the copilot code copied to the clipboard.... It looks like that copilot now wants to be enabled in your GitHub account, which by default is not enabled even if you paste the 8-digit code. To "solve that" go to https://github.com/settings/copilot  after you log in to GitHub→ click enable Copilot big green button → this will land you to  the page to fill in your billing address and payment method from the subscription options. After completing that, if you go to your account →"Billing and plans section" you will see that you successfully subscribed for copilot subscription and that you are currently in 60 days trial period.

![chrome_ypXNoukIfI](https://user-images.githubusercontent.com/19243988/177203359-8516bbf8-3163-4114-aa2f-bcc69a49cb34.png)

And after that if you go to your Visual Studio and click on the copilot icon you will see that now you are signed in, and the plugin now can suggest you code from its AI brain :)

This reminds me of some other cloud provider that wants to fill in your billing/payment details if you want to use their product, even in a trial. Without filling that "thing" just it will not work :)

And don't forget to cancel the subscription if you don't intend to pay for it after the trial period ends.

**Comment by victorcodigo01** [comment]
It's a question really. My case is about I suscribed today and I did't get Copilot.
I show you screenshots. 
What I can do ?


![copilot1](https://user-images.githubusercontent.com/100309538/177217135-766ce311-2dc7-476c-9fac-bd90c96c7c79.png)



![copilot1-2](https://user-images.githubusercontent.com/100309538/177217156-53f48896-8c7c-4a99-9fe0-b090d7f3ecb0.png)



![copilot2](https://user-images.githubusercontent.com/100309538/177217170-1a0e938c-9018-484c-a12d-b8e3b19e60ed.png)



As you can see, in my account of Github I have the suscription activated. But isn't run in Visual Studio Code

**Reply by chesthar** [reply]
Not sure why it's not working for you in VScode. In general, this thread is for Visual Studio 2022 not VScode. But anyway, you can try the to delete the github-copilot folder from C:\Users\{username}\AppData\Local and try to reinstall the plugin again. If it's still not working, check if there is some log in your editor that can hing where is the problem. Could be some communication issue if you are behind a proxy or some firewall that blocks the network communication of the plugin to the outside world.

**Comment by victorcodigo01** [comment]
I didn't found the folder of github-copilot in
C:\Users{username}\AppData\Local
Something happened, cause I probe to uninstall , and reinstall the
extension but without success.

I cancelled my subscription to github copilot as you see in the photo

**Comment by victorcodigo01** [comment]
I log out about my github account from VISUAL STUDIO CODE.

if I install Github Copilot extension don't log in (the las time loged in)
I dont know is about log in

**Reply by kzu** [reply]
This question is about Visual Studio, not VS Code.

**Comment by krzysiek-b** [comment]
Hi,
I'm using latest Visual Studio and copilot extension and seems the hints are not working as expected. When I type and even invoke the "trigger inline suggestion" from the document's icon nothing happens. The hints start to apear when I click on the "open copilot" and I can accept them by TAB.

I was thinking this is due to the code but when I use VSCode or Rider on the same code it works fine.

I'm also using R# but I did the "configuration" proposal from github. I also see they added an option for copilot too.
<img width="424" alt="image" src="https://user-images.githubusercontent.com/6192236/219591800-dee7d650-dfe6-4a51-9dff-a22b554f67ed.png">

I tried reauth the copilot it did not helped.

Any idea what is wrong in VS? Are there any logs I can investigate?

**Reply by ayim** [reply]
[krzysiek-b](https://github.com/krzysiek-b), please shoot me an email at ayim at github dot com.
We are looking to understand the full scope of issues, it would be incredibly helpful if we could do a screenshare and collect some logs together.

**Comment by woeterman94** [comment]
I need some help here to. The browser is not opening and no device token is in my clipboard.

```
GitHub Copilot 1.77.0.0 (v1.77.0.0@1b89dfbba)
GitHub Copilot Agent v1.77.0
[Error] An error occurred for 'RunOperation:SignInInitiateAsync': StreamJsonRpc.RemoteInvocationException: Request signInInitiate failed with message: unable to get local issuer certificate
   at StreamJsonRpc.JsonRpc.<InvokeCoreAsync>d__145`1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at GitHub.Copilot.CopilotClient.<RunOperation>d__20`1.MoveNext()
[Error] An error occurred for 'RunOperation:RunOperation': StreamJsonRpc.RemoteInvocationException: Request signInInitiate failed with message: unable to get local issuer certificate
   at StreamJsonRpc.JsonRpc.<InvokeCoreAsync>d__145`1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at GitHub.Copilot.CopilotClient.<RunOperation>d__20`1.MoveNext()
NotSignedIn
[Error] An error occurred for 'RunOperation:SignInInitiateAsync': StreamJsonRpc.RemoteInvocationException: Request signInInitiate failed with message: unable to get local issuer certificate
   at StreamJsonRpc.JsonRpc.<InvokeCoreAsync>d__145`1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at GitHub.Copilot.CopilotClient.<RunOperation>d__20`1.MoveNext()
[Error] An error occurred for 'RunOperation:RunOperation': StreamJsonRpc.RemoteInvocationException: Request signInInitiate failed with message: unable to get local issuer certificate
   at StreamJsonRpc.JsonRpc.<InvokeCoreAsync>d__145`1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at GitHub.Copilot.CopilotClient.<RunOperation>d__20`1.MoveNext()
NotSignedIn
```

**Reply by ayim** [reply]
@woeterman94, please shoot me an email at ayim at github dot com.
This error is new to me... it would be incredibly helpful if we could do a screenshare and collect some logs together.

**Reply by hg98-park** [reply]
Having silimar problem here.
[DEBUG] [agent] [2023-04-06T04:32:20.857Z] Agent service starting
[DEBUG] [agent] [2023-04-06T04:32:20.903Z] Telemetry initialized
[Error] An error occurred for 'NotifyAgentRestarted': System.Net.Http.HttpRequestException: 이 요청을 보내는 동안 오류가 발생했습니다. ---> System.Net.WebException: 기본 연결이 닫혔습니다. 예기치 않게 연결이 닫혔습니다.
   위치: System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   위치: System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 내부 예외 스택 추적의 끝 ---
   위치: System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   위치: System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   위치: GitHub.Copilot.PanicEndpointSubmitter.<PostToEndpointAsync>d__11.MoveNext()
--- 예외가 throw된 이전 위치의 스택 추적 끝 ---
   위치: System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   위치: System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   위치: GitHub.Copilot.PanicEndpointSubmitter.<PostToPanicEndpointAsync>d__10.MoveNext()
--- 예외가 throw된 이전 위치의 스택 추적 끝 ---
   위치: System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   위치: System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   위치: GitHub.Copilot.CopilotClient.<>c__DisplayClass18_0.<<NotifyAgentRestarted>b__0>d.MoveNext()
--- 예외가 throw된 이전 위치의 스택 추적 끝 ---
   위치: System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   위치: System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   위치: Microsoft.VisualStudio.Shell.VsTaskLibraryHelper.<>c__DisplayClass35_0.<<FileAndForget>b__0>d.MoveNext()
[Error] An error occurred for 'NotifyAgentRestarted': System.Net.Http.HttpRequestException: 이 요청을 보내는 동안 오류가 발생했습니다. ---> System.Net.WebException: 기본 연결이 닫혔습니다. 예기치 않게 연결이 닫혔습니다.
   위치: System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   위치: System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- 내부 예외 스택 추적의 끝 ---
   위치: System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   위치: System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   위치: GitHub.Copilot.PanicEndpointSubmitter.<PostToEndpointAsync>d__11.MoveNext()
--- 예외가 throw된 이전 위치의 스택 추적 끝 ---
   위치: System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   위치: System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   위치: GitHub.Copilot.PanicEndpointSubmitter.<PostToPanicEndpointAsync>d__10.MoveNext()
--- 예외가 throw된 이전 위치의 스택 추적 끝 ---
   위치: System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   위치: System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   위치: GitHub.Copilot.CopilotClient.<>c__DisplayClass18_0.<<NotifyAgentRestarted>b__0>d.MoveNext()
--- 예외가 throw된 이전 위치의 스택 추적 끝 ---
   위치: System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   위치: System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   위치: Microsoft.VisualStudio.Shell.VsTaskLibraryHelper.<>c__DisplayClass35_0.<<FileAndForget>b__0>d.MoveNext()
[Error] An error occurred for 'RunOperation:SignInInitiateAsync': StreamJsonRpc.RemoteInvocationException: Request signInInitiate failed with message: self signed certificate in certificate chain
   위치: StreamJsonRpc.JsonRpc.<InvokeCoreAsync>d__145`1.MoveNext()
--- 예외가 throw된 이전 위치의 스택 추적 끝 ---
   위치: System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   위치: System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   위치: GitHub.Copilot.CopilotClient.<RunOperation>d__20`1.MoveNext()
[DEBUG] [agent] [2023-04-06T04:32:30.436Z] Agent service starting
[DEBUG] [agent] [2023-04-06T04:32:30.494Z] Telemetry initialized
[Error] An error occurred for 'RunOperation:RunOperation': StreamJsonRpc.RemoteInvocationException: Request signInInitiate failed with message: self signed certificate in certificate chain
   위치: StreamJsonRpc.JsonRpc.<InvokeCoreAsync>d__145`1.MoveNext()
--- 예외가 throw된 이전 위치의 스택 추적 끝 ---
   위치: System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   위치: System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   위치: GitHub.Copilot.CopilotClient.<RunOperation>d__20`1.MoveNext()
[DEBUG] [agent] [2023-04-06T04:32:35.402Z] Agent service starting
[DEBUG] [agent] [2023-04-06T04:32:35.452Z] Telemetry initialized
NotSignedIn

**Reply by ayim** [reply]
@hg98-park, from reading the log, you should be able to fix this in Visuial Studio by going to Tools > Options > GitHub Copilot > and set Allow non-trusted certificates automatically to 'True'

<img width="813" alt="image" src="https://github.com/community/community/assets/10951869/4f5599cb-03a4-4149-8ab2-779a6474a53e">

**Comment by furman87** [comment]
I'm on VS2022 17.6.5 and Copilot 1.94.0.0 (v1.94.0.0@6586dd2df). I've never gotten an activation code, and this is what I see in the Copilot output window when I open a file:

GitHub Copilot 1.94.0.0 (v1.94.0.0@6586dd2df)
GitHub Copilot Agent v1.94.0
[DEBUG] [agent] [2023-07-29T23:00:59.728Z] Agent service starting
[DEBUG] [agent] [2023-07-29T23:00:59.840Z] Telemetry initialized
Auth Status: OK
Successfully authorized GitHub Copilot.
[INFO] Attempted sign in activeAccount https://github.com/2974590

It's obviously trying to sign into my github account, but it appears as if it already authenticated, so that seems strange. Since I've never gotten the activation code, Copilot isn't working for me.

I have tried uninstalling and reinstalling the Copilot extension but that hasn't worked. I have deleted the %LOCALAPPDATA%\github-copilot directory after uninstalling, then reinstalled the extension. No dice, same output as above every time.

If I hover over the Copilot icon in VS2022, I see this:

![image](https://github.com/community/community/assets/2974590/c152646b-08d6-408a-bf2d-b2d2cf70a775)

And if I click on it, these are my menu options (no menu item for logging in or out)

![image](https://github.com/community/community/assets/2974590/ee85785f-4609-4488-abe9-edc6cc5317f3)

Anybody seen this before or had it happen to them?

**Reply by ayim** [reply]
@furman87 the mismatch between the extension status (i.e. Copilot has not been started) and the actual status was resolved several versions ago.

RE: Activation code, that's an old authentication process that's no longer used. It appears to me like you're signed in to Copilot properly and have an entitlement.

Make it or break it question: are you seeing grey text code predictions that are larger than one line?

**Comment by ObaidNiazi** [comment]
I have the same problem with visual 2022 pro, it says I am authorized but never seems to suggest code any longer having used it successfully for couple of months its stopped working could it be because of using VisualVSN?  at startup visual studio prompts to choose either git or subversion but that should only affect repository/versioning not the copilot or am I mistaken? Please help
![image](https://github.com/community/community/assets/44118322/70122d9e-0f9d-42f1-a418-6a3ce12822ba)

**Reply by ayim** [reply]
@ObaidNiazi your intuition is correct - your version control should not be affecting the GitHub Copilot code predictions.
That's weird that it's stopped working.... would you mind:

1) Filing a feedback issue in VS Feedback
2) Sending me an email at ayim@microsoft.com so that we can setup a call and investigate?

Thanks in advance,
Aaron

<img width="459" alt="image" src="https://github.com/community/community/assets/10951869/782fc71f-cc77-4382-a10f-6c67809c48f4">

**Comment by grsh50** [comment]
In my case, adding the github account again helped.
